nohup: ignoring input
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu line=58 error=2 : out of memory
hvac
hvac 167.332021435
{'hvac': 167.33202143476146}
{'hvac': 201.59651892300334}
fridge
fridge 28.1751699348
{'fridge': 28.17516993478532}
{'hvac': 201.59651892300334, 'fridge': 340.45910569799219}
dr
dr 66.9294258049
{'dr': 66.929425804880026}
{'hvac': 201.59651892300334, 'fridge': 340.45910569799219, 'dr': 1048.8100582089096}
dw
dw 15.4395102012
{'dw': 15.43951020120803}
{'hvac': 201.59651892300334, 'fridge': 340.45910569799219, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109}
mw
mw 9.09643141771
{'mw': 9.0964314177102956}
{'hvac': 201.59651892300334, 'fridge': 340.45910569799219, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109, 'mw': 1048.7359408730861}
hvac 301.730521573
fridge 70.3369294413
hvac 167.305203973
dr 66.9294258049
hvac 167.294576401
dw 15.4395102012
hvac 399.071829482
mw 9.09705703876
fridge 40.8507394945
hvac 119.49509534
fridge 188.394848208
dr 66.9294258049
fridge 28.1767665803
dw 15.4395102012
fridge 62.4795127983
mw 9.09690026449
mw 21.4554692619
hvac 151.762863926
mw 9.09707997968
fridge 28.1869912998
mw 9.09707779131
dr 66.9294258049
mw 9.09707997968
dw 15.4395102012
Traceback (most recent call last):
  File "rnn-tree-greedy.py", line 225, in <module>
    new_error = disagg(cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 184, in disagg
    pred, gt = disagg_fold(cur_fold, cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 136, in disagg_fold
    pred = a(*params)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 90, in forward
    self.preds[appliance] = getattr(self, "Appliance_" + str(appliance))(agg_current)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 48, in forward
    pred, hidden = self.rnn(x, None)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/rnn.py", line 169, in forward
    output, hidden = func(input, self.all_weights, hx)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 385, in forward
    return func(input, *fargs, **fkwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 328, in _do_forward
    flat_output = super(NestedIOFunction, self)._do_forward(*flat_input)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 350, in forward
    result = self.forward_extended(*nested_tensors)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 294, in forward_extended
    cudnn.rnn.forward(self, input, hx, weight, output, hy)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/backends/cudnn/rnn.py", line 291, in forward
    fn.reserve = torch.cuda.ByteTensor(reserve_size.value)
RuntimeError: cuda runtime error (2) : out of memory at /opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu:58
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu line=58 error=2 : out of memory
hvac
hvac 167.327096435
{'hvac': 167.32709643486191}
{'hvac': 201.59058543324488}
fridge
fridge 28.1733298654
{'fridge': 28.173329865396177}
{'hvac': 201.59058543324488, 'fridge': 340.43687092957379}
dr
dr 66.9294258049
{'dr': 66.929425804880026}
{'hvac': 201.59058543324488, 'fridge': 340.43687092957379, 'dr': 1048.8100582089096}
dw
dw 15.4395102012
{'dw': 15.43951020120803}
{'hvac': 201.59058543324488, 'fridge': 340.43687092957379, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109}
mw
mw 9.09709557186
{'mw': 9.0970955718588034}
{'hvac': 201.59058543324488, 'fridge': 340.43687092957379, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109, 'mw': 1048.8125118154519}
hvac 295.251125245
fridge 71.2013669237
hvac 167.300335118
dr 66.9294258049
hvac 167.300686564
dw 15.4395102012
hvac 391.690801429
mw 9.09706747987
fridge 53.0390565389
hvac 131.986500528
fridge 188.554034962
dr 66.9294258049
fridge 28.17509933
dw 15.4395102012
fridge 62.6144226254
mw 9.09543154036
dw 15.4395102012
hvac 167.300857915
dw 15.4395102012
fridge 28.2018950017
dw 15.4395102012
dr 66.9294258049
dw 15.4395102012
mw 9.09707779131
Traceback (most recent call last):
  File "rnn-tree-greedy.py", line 225, in <module>
    new_error = disagg(cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 184, in disagg
    pred, gt = disagg_fold(cur_fold, cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 136, in disagg_fold
    pred = a(*params)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 90, in forward
    self.preds[appliance] = getattr(self, "Appliance_" + str(appliance))(agg_current)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 48, in forward
    pred, hidden = self.rnn(x, None)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/rnn.py", line 169, in forward
    output, hidden = func(input, self.all_weights, hx)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 385, in forward
    return func(input, *fargs, **fkwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 328, in _do_forward
    flat_output = super(NestedIOFunction, self)._do_forward(*flat_input)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 350, in forward
    result = self.forward_extended(*nested_tensors)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 294, in forward_extended
    cudnn.rnn.forward(self, input, hx, weight, output, hy)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/backends/cudnn/rnn.py", line 291, in forward
    fn.reserve = torch.cuda.ByteTensor(reserve_size.value)
RuntimeError: cuda runtime error (2) : out of memory at /opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu:58
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu line=58 error=2 : out of memory
hvac
hvac 167.326649537
{'hvac': 167.32664953693313}
{'hvac': 201.59004702424232}
fridge
fridge 28.1755420083
{'fridge': 28.175542008265381}
{'hvac': 201.59004702424232, 'fridge': 340.46360170651917}
dr
dr 66.9294258049
{'dr': 66.929425804880026}
{'hvac': 201.59004702424232, 'fridge': 340.46360170651917, 'dr': 1048.8100582089096}
dw
dw 15.4395102012
{'dw': 15.43951020120803}
{'hvac': 201.59004702424232, 'fridge': 340.46360170651917, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109}
mw
mw 9.09709557186
{'mw': 9.0970955718588034}
{'hvac': 201.59004702424232, 'fridge': 340.46360170651917, 'dr': 1048.8100582089096, 'dw': 1048.8099434418109, 'mw': 1048.8125118154519}
hvac 293.862734274
fridge 71.544039135
hvac 167.298459106
dr 66.9294258049
hvac 167.299573971
dw 15.4395102012
hvac 383.315609009
mw 9.0970433348
fridge 52.8269599987
hvac 131.884945173
fridge 188.095914672
dr 66.9294258049
fridge 28.1271449355
dw 15.4395102012
fridge 62.6910338462
mw 9.09467545823
dw 15.4395102012
hvac 167.299453433
dw 15.4395102012
fridge 28.0742668258
dw 15.4395102012
dr 66.9294258049
dw 15.4395102012
mw 9.09707779131
Traceback (most recent call last):
  File "rnn-tree-greedy.py", line 225, in <module>
    new_error = disagg(cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 184, in disagg
    pred, gt = disagg_fold(cur_fold, cell_type, hidden_size, num_layers, bidirectional, lr, num_iterations, p)
  File "rnn-tree-greedy.py", line 136, in disagg_fold
    pred = a(*params)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 90, in forward
    self.preds[appliance] = getattr(self, "Appliance_" + str(appliance))(agg_current)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "rnn-tree-greedy.py", line 48, in forward
    pred, hidden = self.rnn(x, None)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py", line 325, in __call__
    result = self.forward(*input, **kwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/modules/rnn.py", line 169, in forward
    output, hidden = func(input, self.all_weights, hx)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 385, in forward
    return func(input, *fargs, **fkwargs)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 328, in _do_forward
    flat_output = super(NestedIOFunction, self)._do_forward(*flat_input)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/autograd/function.py", line 350, in forward
    result = self.forward_extended(*nested_tensors)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/nn/_functions/rnn.py", line 294, in forward_extended
    cudnn.rnn.forward(self, input, hx, weight, output, hy)
  File "/zf14/yj9xs/anaconda3/lib/python3.6/site-packages/torch/backends/cudnn/rnn.py", line 291, in forward
    fn.reserve = torch.cuda.ByteTensor(reserve_size.value)
RuntimeError: cuda runtime error (2) : out of memory at /opt/conda/conda-bld/pytorch_1512386481460/work/torch/lib/THC/generic/THCStorage.cu:58
svm-2.sh: line 8: 10442 Terminated              CUDA_VISIBLE_DEVICES=2 python rnn-tree-greedy.py GRU 100 2 True 1 800 0
svm-2.sh: error reading input file: Stale file handle
